discovery_engine_core/web-api/Dockerfile.ingestion

```diff
@@ -1,11 +0,0 @@
-FROM debian:bookworm-slim
-
-WORKDIR /service
-
-COPY ./ ./
```

Comment:
This doesn't do what it seems to do:

- it seems to copy assets
- it will produce a empty asset dir as it won't follow symlinks outside of the build root
  - (or with the changes to just make the `assets` dir a symlink the whole dir is skipped)
  - similary you can't copy ../../.........v0003/* as .. is outside the build root

it also copies a lot of things we do not want to copy